### DISCUSSION TITLE
Prior Monte Carlo Sweep vs IES iteration zero

### DISCUSSION THREAD

**Discussion by laat0003** [usgs/pestpp#261] [Q&A] [answered]
Hi,

I noticed that when I run a prior Monte Carlo through SWP I get more failed runs than when I use that same parameter ensemble in iteration 0 with IES. Is this due to differences in the default run manager settings for each executable or is it more than that? 

Cheers,

**Comment by jtwhite79** [comment, marked as answer]
That would be my guess.  Can you see why the runs were marked as failed in the rmr file?  Might be some clues in there about average runtimes and run mgr tolerances....

**Reply by laat0003** [reply]
Hmm, now why didn't I think of that. Smort.

If I call SWP and then IES using the same control file in the same folder then the rmr file gets overwritten correct? Curse my OCD for making me want to minimize the number of files and folders.
👍 

Cheers for the quick response.
